_config/acc_staging.yml

```diff
@@ -1,4 +1,4 @@
-id: acc
+id: acc_staging
```

Comment:
I see the re-introduction here. These changes should be removed from the commit history. Otherwise, you've introduced a commit here which, if someone were to revert master back to, would break the deployment pipeline. The point of the deliberate git flow is to only have the commit history reflect production-ready code at all times.